src/components/Asset/AssetActions/Pool/Graph/index.tsx

```diff
@@ -52,26 +52,20 @@ export default function Graph({
 
     const timestamps = poolSnapshots.map((item) => {
       const date = new Date(item.date * 1000)
-      return `${date.toLocaleDateString(locale)} ${date.toLocaleTimeString(
-        locale,
-        { hour: '2-digit', minute: '2-digit' }
-      )}`
+      return `${date.toLocaleDateString(locale)}`
     })
 
-    let baseTokenLiquidityCumulative = '0'
     const liquidityHistory = poolSnapshots.map((item) => {
       const conversionSpotPrice = prices[currency.toLowerCase()]
-      baseTokenLiquidityCumulative = new Decimal(baseTokenLiquidityCumulative)
-        .add(item.baseTokenLiquidity)
-        .mul(2) // double baseTokenLiquidity as we have 50/50 weight
+
+      // the tab is name liquidity, why convert to fiat?
+      const convertedLiquidity = new Decimal(item.baseTokenLiquidity)
```

Comment:
yeah good point, as we output the TVL number further down I found it more valuable to also show the graph in that user currency but didn't adapt the variable names. So to be more precise we should use TVL as the term here so this could be `const tvl = ...`.

That also raises a good point that we might want to switch some more labels for our pool sections. Like, "Your Liquidity" could become "Your Value Locked"